### DISCUSSION TITLE
How to use Pinia plugin persist in nuxt3

### DISCUSSION THREAD

**Discussion by LY-Net** [vuejs/pinia#1301] [Help and Questions]
### Reproduction

localstorage is not defined

### Steps to reproduce the bug

How to use Pinia plugin persist in nuxt3 and Solve the problem of localstorage is not defined

### Expected behavior

How to use Pinia plugin persist in nuxt3 and Solve the problem of localstorage is not defined

### Actual behavior

How to use Pinia plugin persist in nuxt3 and Solve the problem of localstorage is not defined

**Comment by DidierHoarau** [comment]
Hi,

Unsure if my problem was similar to yours but I did encounter the problem that:
- I was trying to use localStorage in a Pinia store
- I got the "localstorage is not defined" error

Actually I think it's because of Server Side Rendering.

On first load the code is executed in the server, in that case localStorage doesn't exist. I solved my issue by disabling SSR in nuxt.config.ts
I didn't need SSR on my side. Unsure if there is a better way.
